scripts/apps/tansa/index.ts

```diff
@@ -28,7 +31,11 @@ document.write(`
             $rootScope.$broadcast('tansa:end', isCancelled);
         }
     </script>
-`);
+`;
+
+// workaround to avoid nggettext_extract parsing error
+tansaScript = tansaScript.replace('<baseUrl', baseUrl);
```

Comment:
I think you are missing a `>` in the string.

Reply:
Fixed

Reply:
![screenshot_7](https://user-images.githubusercontent.com/2076953/51601650-40903180-1f05-11e9-96c3-ace4f05832b0.png)

Did I mention that these "Fixed" and "Done" comments are annoying? :smiley: Use the resolve button.